packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -941,6 +941,15 @@ const schema: SchemaType<"Users"> = {
     canCreate: 'guests',
     hidden: true,
   },
+  frontpageSelectedTab: {
+    type: String,
+    optional: true,
+    nullable: true,
+    canRead: userOwns,
+    canUpdate: [userOwns, 'sunshineRegiment', 'admins'],
```

Comment:
I think this probably shouldn't include `sunshineRegiment`; it's not moderation-related

Reply:
Well... we'd want to not include `lesswrong` in the tab name, in that case.